### ISSUE TITLE
SFZ parsing incorrect

### ISSUE THREAD

**Issue by StratusFearMe21** [arduano/xsynth#18]
For whatever reason, when I attempt to load the CFaz Keys IV in [wasabi](https://github.com/ardurano/wasabi), it simply doesn't load and it is silent when I load a MIDI file. And when I tried to run the example in this repository for the renderer, I got this error
```
called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', /home/isaacm/xsynth/core/src/soundfont/audio.rs:47:42
```
So, I added some `dbg!()` statements around where the error was and as it turns out the contents of the `PathBuf` it was trying to load was
```
"/home/isaacm/CFazKeys-4/1. CFaz Keys IV - Concert Presets/../.PianoSample/CFAZIV_093.wav pan=29 key=93"
```

Other than that, I'm super happy to see a MIDI synthesizer with KMC quality that works natively on Linux at all! Very cool project

**Comment by Kaydax** [comment]
This isn't a Linux issue but instead a general parsing issue when it comes to sfz's that put multiple parameters on a single line. SFZ loading works on Linux as I and MBMS both use Linux daily, so the issue is more about the way we parse the SFZ and not a Linux exclusive issue

**Comment by StratusFearMe21** [comment]
I thought so, I just wasn't sure.

**Comment by Kaydax** [comment]
Should be fixed with #19
